examples/android/sync/build.gradle

```
@@ -4,18 +4,24 @@ buildscript {
     repositories {
         google()
         jcenter()
+        maven { // used for realm snapshot releases
+            url 'https://oss.sonatype.org/content/repositories/snapshots/'
+        }
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:7.0.3'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath "io.realm:realm-gradle-plugin:10.8.1"
+        classpath "io.realm:realm-gradle-plugin:10.9.0-SNAPSHOT"
```

Once 10.9.0 comes out, we'll want to change this library reference to point to the production release. It's OK to leave the other snapshot release stuff in here because we'll need it sporadically in the future for testing new releases anyway.

```suggestion
        classpath "io.realm:realm-gradle-plugin:10.9.0"
```